usecases/linearFunction/test_optimize.py

```diff
@@ -34,33 +37,48 @@ def __init__(self, yaml_file_metadata, yaml_file_data):
         with open(yaml_file_data, "r") as f:
             data = yaml.load(f, Loader=yaml.FullLoader)
 
+        self.linearModelParameters = []
         for idx, a in enumerate(all_a):
             data_f = np.asarray(data[idx]['f'])
             data_df = np.asarray(data[idx]['df'])
             assert(a == data[idx]['a'])
             single_model_error = LinearModelError(x_function, x_derivative, data_f, data_df, a)
             # read in the parameters that are given in the experiment (in this case constant offset a)
             parameter = single_model_error.get_parameter_dict()
-            # add parameters that are model parameters (not given in the experimental data)
-            parameter.define("b")
+            # store the parameter list here to access it later, e.g. in the evaluation of the likelihood
+            self.linearModelParameters.append(parameter)
+            # add the parameter list with the model error to the joint latent list
             self.add(single_model_error, parameter)
 
-        # define one shared latent parameter (only one parameter b for all linear models)
-        self.latent.add_by_name('b')
+    def latent_as_tensor_variable(self, model, prior_pymc3):
```

Comment:
I think, with your latest commit, you do not need this method anymore.

Reply:
yes, thanks, I will delete that.